plugins/modules/current_container_facts.py

```diff
@@ -54,37 +54,42 @@
         ansible_module_container_type:
             description:
               - The detected container environment.
-              - Contains an empty string if no container was detected.
-              - Otherwise, will be one of C(docker), C(azure_pipelines), or C(github_actions).
+              - Contains an empty string if no container was detected, or a non-empty string identifying the container environment.
               - C(github_actions) is supported since community.docker 2.4.0.
+              - C(podman) is supported since community.docker 3.3.0.
             returned: always
             type: str
             choices:
               - ''
               - docker
               - azure_pipelines
               - github_actions
+              - podman
 '''
 
 import os
+import re
 
 from ansible.module_utils.basic import AnsibleModule
 
 
 def main():
     module = AnsibleModule(dict(), supports_check_mode=True)
 
-    path = '/proc/self/cpuset'
+    cpuset_path = '/proc/self/cpuset'
+    mountinfo_path = '/proc/self/mountinfo'
+
     container_id = ''
     container_type = ''
 
-    if os.path.exists(path):
+    contents = None
+    if os.path.exists(cpuset_path):
         # File content varies based on the environment:
         #   No Container: /
```

Comment:
This comment is not reflecting the changes. But i'm hesitant whether it should be updated or removed.

Reply:
I extended the comment in 23a07989327966b734d26a8521e3c8698b3595d0, I hope my extension is accurate enough :)

Reply:
/resolved